src/components/Table/EditableDraggableTable.js

```diff
@@ -205,6 +206,10 @@ class EditableDraggableTable extends React.Component {
       result = value;
     }
 
+    if (!editable && readValueFormatter) {
+      result = readValueFormatter(result);
+    }
+
```

Comment:
If implemented using static addendum, the logic would need to be moved to a higher level while affecting other flows and so potentially introducing new bugs. So I decided to go this safer way.

One more idea: we can keep using addendum as it is and _optionally_ assigning a function to it (instead of recent string). The `EdditableDraggableTable` would check it's type and act accordingly. But such reusing of single property for two different types introduces more confusion, imo.

Reply:
Ok, we can leave this for later. Can you please create an issue to unify these two approaches?

Yes better to rewrite it without any magic and use just the functional approach.